pyrightconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "reportAttributeAccessIssue": false
```

Comment:
So this is a super annoying lint, that produces unusable errors imo.

Everytime I try to use `Prefetch` with `to_attr`, I can't use the attr that is assigned to the object, as I otherwise get this type error. So I'd rather just disables it.